static/src/stylesheets/module/business/_stocks.scss

```diff
@@ -0,0 +1,170 @@
+$stocks-icon-width: 14px;
+$stocks-icon-height: 10px;
+$closed-colour: #767676;
+
+.stocks__stocks-container {
+    @include clearfix;
+
+    clear: left;
+    margin-top: $gs-baseline * 1.5;
+    margin-bottom: $gs-baseline;
+    overflow-x: scroll;
+    white-space: nowrap;
+    -webkit-overflow-scrolling: touch;
+
+    &::-webkit-scrollbar {
+        display: none;
+    }
+}
+
+.stocks__divider {
+    width: 50%;
+
+    @include mq(leftCol) {
+        border-top: 1px dotted guss-colour(neutral-3, $pasteup-palette);
+    }
+
+    @include mq(wide) {
+        .has-page-skin & {
+            border-top: 0;
+        }
+    }
+}
+
+.stocks__change:before {
+    content: '';
+    width: $stocks-icon-width;
+    height: $stocks-icon-height;
+    margin-right: 3px;
+}
+
+.stocks__stock-value--negative {
+    .stocks__change {
+        color: colour(status-negative);
+
+        &:before {
+            @include icon(market-down, false);
+        }
+    }
+}
+
+.stocks__stock-value--positive {
+    .stocks__change {
+        color: colour(status-positive);
+
+        &:before {
+            @include icon(market-up, false);
+        }
+    }
+}
+
+.stocks__stock-value--level {
+    .stocks__change {
+        color: colour(status-neutral);
+
+        &:before {
+            @include icon(market-same, false);
+        }
+    }
+}
```

Comment:
To be a bit more DRY you could refactor these modifiers into a mixin and pass in the keys:

```scss
@mixin stock($change, $colour, $icon) {
.stocks__stock-value--#{change} {
    .stocks__change {
        color: colour($colour);

        &:before {
            @include icon($icon, false);
        }
    }
}
}

@include stock('positive', status-positive, market-up);
@include stock('negative', status-negative, market-down);
@include stock('level', status-neutral, market-same);